public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "WebVR Studio",
+  "name": "Build WebVR experiences with WebVR Studio.",
```

Comment:
Should we change these to project strings like in package.json? I've not dealt with a manifest.json before - where can i read more on this?